clippy_lints/src/methods/mod.rs

```diff
@@ -1102,6 +1102,40 @@ declare_clippy_lint! {
     "Check for offset calculations on raw pointers to zero-sized types"
 }
 
+declare_clippy_lint! {
+    /// **What it does:** Checks for `FileType::is_file()`.
+    ///
+    /// **Why is this bad?** When people testing a file type with `FileType::is_file`
+    /// they are testing whether a path is something they can get bytes from. But
+    /// `is_file` doesn't cover special file types in unix-like systems. and not covering
```

Comment:
```suggestion
    /// `is_file` doesn't cover special file types in unix-like systemsand does not cover symlinks
```

Reply:
Missing a comma here.
```suggestion
    /// `is_file` doesn't cover special file types in unix-like systems, and doesn't cover
```